.github/workflows/e2e.yaml

```
@@ -133,6 +133,14 @@ jobs:
           done
         timeout-minutes: 1
 
+      - name: Check that the daemon-set for device plugin is running
+        run: |
+          DS_NAME=$(kubectl get ds -l oot.node.kubernetes.io/module.name=ooto-ci -o name)
+          until kubectl get -o json $DS_NAME  | jq -e '.status.numberReady | contains(1)'; do 
```

I think we can have it in one command only:
```suggestion
          until kubectl get ds -l oot.node.kubernetes.io/module.name=ooto-ci,oot.node.kubernetes.io/role=device-plugin -o json | jq -e '.items[] | select(.status.numberReady == 1)''; do 
```

---

The selector should be like the new label `kmm.node.kubernetes.io/role` instead of `oot.node.kubernetes.io`, right?